backend/src/main/java/com/cgi/retrospecto/backend/poker/domain/Room.java

```diff
@@ -0,0 +1,92 @@
+package com.cgi.retrospecto.backend.poker.domain;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Room {
+    private int id;
+    private String author;
+    private List<Story> stories = new ArrayList<>();
+    private LocalDateTime lastActionSubmittedTime;
+    private Set<String> users = new HashSet<>();
+    private Story selectedStory;
+
+    public Room() {
+    }
+
+    public List<Integer> getStoriesIdsList() {
+        return stories.stream().map(s -> s.getId()).collect(Collectors.toList());
```

Comment:
As IntelliJ suggests this can be replaced with:
return stories.stream().map(Story::getId).collect(Collectors.toList());

Reply:
Done.